src/main/java/com/valstro/plugin/validator/FieldValidator.java

```diff
@@ -90,4 +114,8 @@ private boolean typeIsAClassReference(Class<?> clazz) {
         return holder.isExist(clazz.getSimpleName());
     }
 
+    private boolean typeIsAnEnum(Class<?> clazz) {
```

Comment:
NIT: Is this helper really needed?

Reply:
I agree with u and adding screenshots now

Reply:
I removed the method and moved all the enum validation implementation to his own method